openshift-test-utils/src/main/java/io/openshift/booster/test/OpenShiftTestAssistant.java

```diff
@@ -176,4 +176,29 @@ public void awaitPodReadinessOrFail(Predicate<Pod> filter) {
                                                   }
         );
     }
+
+    public void scale(final int replicas) {
+        System.out.println(String.format("Scaling replicas from %s to %s.", deploymentConfig().getStatus()
+                .getAvailableReplicas(), replicas));
+
+        this.client
+                .deploymentConfigs()
+                .inNamespace(this.project)
+                .withName(this.applicationName)
+                .scale(replicas);
+
+        await().atMost(5, TimeUnit.MINUTES)
+                .until(() -> deploymentConfig()
+                        .getStatus()
+                        .getAvailableReplicas() == replicas
```

Comment:
Right, that's exactly how we'd do it.

I don't really care about the `deploymentConfig` method. It doesn't harm anything, but it isn't particularly helpful either.